swc/private/swc.bzl

```diff
@@ -66,6 +66,10 @@ https://docs.aspect.build/rulesets/aspect_rules_js/docs/js_library#data for more
     "root_dir": attr.string(
         doc = "a subdirectory under the input package which should be consider the root directory of all the input files",
     ),
+    "experimental_emit_isolated_dts": attr.bool(
```

Comment:
Did you consider whether or not this ought to have the `experimental_` prefix? When swc changes their flag, will we also change our attribute name? That will be breaking, right?

Reply:
We've never had an attribute prefixed with `experimental_` before, and never wrote that it's not public API. I guess it's a fine convention, just surprised that we invent a new one so many years in.

Perhaps it should be exposed as a separate rule instead? Doesn't need to mean copy-paste of code, probably 95% reused in a lib.

Reply:
I could also put EXPERIMENTAL in the docs I guess, but I don't want this to be considered a stable public API at this point. I guess I was partially just following the `jsc.experimental.emitIsolatedDts` swc property 🤷